secboot/encrypt_sb.go

```diff
@@ -94,3 +94,15 @@ func AddRecoveryKey(key EncryptionKey, rkey RecoveryKey, node string) error {
 func (k RecoveryKey) String() string {
 	return sb.RecoveryKey(k).String()
 }
+
+// EnsureRecoveryKey makes sure the encrypted block devices have a recovery key.
+// XXX what is the right signature for this?
+func EnsureRecoveryKey(fdeDir string) (RecoveryKey, error) {
```

Comment:
We usually pass device nodes and key paths explicitly, so perhaps it should take something like
```
deviceToKey map[string]string
```
which would be:
```
/dev/.../ubuntu-data: /var/lib/snapd/device/fde/recovery.key
/dev/.../ubuntu-save: /var/lib/snapd/device/fde/reinstall.key
```

and similar for RemoveRecoveryKeys. Internally we can call snap-fde-keymgr with the right flags for each device.